balancer/catalyst/catalyst_balancer.go

```diff
@@ -1,5 +1,7 @@
 package catalyst
```

Comment:
Probably `catalyst_balancer` yeah?

Reply:
The Golang way is `catalyst.Balancer` right?

Reply:
"Folder names and package names should match" is a convention but stuff violates it all the time like `"github.com/peterbourgon/ff/v3"` being imported like that but used as `ff`. This defaulting to being imported as `catalyst` is weird to me because lots of things are Catalyst. Weeeee could call it `catabalancer` as an homage to the Istanbul cats? 🐈

Reply:
🐱 sold

Reply:
done!